docs/index.rst

```diff
@@ -27,6 +27,7 @@ torchgeo
 
    tutorials/getting_started
    tutorials/benchmarking
+   tutorials/trainer_tutorial
```

Comment:
This filename is kind of redundant, let's just call it `tutorials/trainers.ipynb`

Reply:
renaming now